<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TutorialClass</class>
 <widget class="QMainWindow" name="TutorialClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tutorial</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="regisTab">
     <attribute name="title">
      <string>Registration</string>
     </attribute>
     <widget class="QWidget" name="regisWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>371</width>
        <height>201</height>
       </rect>
      </property>
      <widget class="QLabel" name="imgLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Img</string>
       </property>
      </widget>
      <widget class="QLabel" name="birthLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Birthday</string>
       </property>
      </widget>
      <widget class="QLabel" name="passwordLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="birthdayEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="fileEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse ..</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelRegisButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="myLabel">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>121</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="nameLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="loginTab">
     <attribute name="title">
      <string>Log in </string>
     </attribute>
     <widget class="QWidget" name="loginWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>371</width>
        <height>201</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="passwordEdit_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="loginButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Log in</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelButon">
       <property name="geometry">
        <rect>
         <x>174</x>
         <y>100</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameEdit_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="myLabel_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="passwordLabel_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="nameLabel_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="refillButton">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>100</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>refill</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>birthdayEdit</tabstop>
  <tabstop>fileEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelRegisButton</tabstop>
  <tabstop>nameEdit_2</tabstop>
  <tabstop>passwordEdit_2</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>refillButton</tabstop>
  <tabstop>cancelButon</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="tutorial.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fileEdit</sender>
   <signal>clicked()</signal>
   <receiver>TutorialClass</receiver>
   <slot>fileAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelRegisButton</sender>
   <signal>clicked()</signal>
   <receiver>TutorialClass</receiver>
   <slot>cancelAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>TutorialClass</receiver>
   <slot>saveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>TutorialClass</receiver>
   <slot>loginAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refillButton</sender>
   <signal>clicked()</signal>
   <receiver>TutorialClass</receiver>
   <slot>refillLoginAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButon</sender>
   <signal>clicked()</signal>
   <receiver>TutorialClass</receiver>
   <slot>cancelAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileAction()</slot>
  <slot>cancelAction()</slot>
  <slot>saveAction()</slot>
  <slot>loginAction()</slot>
  <slot>refillLoginAction()</slot>
 </slots>
</ui>
